### DISCUSSION TITLE
Implementing the "clone building under cursor" functionality

### DISCUSSION THREAD

**Discussion by ravi-donepudi** [dalerank/Akhenaten#133] [Q&A]
My favorite quality of life improvement that Julius/Augustus adds is the "clone building under cursor" functionality. I.e. if you hover over a building and press q (or the appropriate chosen hotkey) it picks the building and lets you build it rather than having to go to the menu and pick.

I have downloaded the latest build of Akhenaten and it appears this feature is not implemented yet. Could someone please suggest how one would go about implementing such a feature? I am happy to try and help implement this feature so that it can be included in an upcoming release. Thank you! :)

Update: Crossposted this to the [citybuilding tools repo](https://github.com/bvschaik/citybuilding-tools/issues/12) for better reach.

**Comment by dalerank** [comment]
Sure, no problem to add.

**Comment by ravi-donepudi** [comment]
Thank you. That is great to hear. Please let me know if I can help in anyway.
